cosmos/launchpad/construction_metadata.go

```diff
@@ -0,0 +1,35 @@
+package launchpad
+
+import (
+	"context"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+func (l Launchpad) ConstructionMetadata(ctx context.Context, r *types.ConstructionMetadataRequest) (*types.ConstructionMetadataResponse, *types.Error) {
+	if r.NetworkIdentifier == nil {
+		return nil, ErrInvalidRequest
+	}
+
+	if r.Options == nil {
+		return nil, ErrInvalidRequest
+	}
+
+	addr := r.Options[OptionAddress]
+	addrString := addr.(string)
+	accRes, _, err := l.cosmos.Auth.AuthAccountsAddressGet(ctx, addrString)
+	if err != nil {
+		return nil, ErrInvalidAddress
```

Comment:
this can be a network related error as well. I think in order to determine if this is an invalid address, we may need to unlock the 2nd return value of `AuthAccountsAddressGet()` and check for an error response returned from the API.

Reply:
True, it's better if we can come up with a way to return the `err` rather than checking multiple errors. This will be the case also for couple of future apis

Reply:
Yep, maybe for now we should change this with the network error and later we start gracefully handling error responses for all APIs.